tensorboard/data/server/DEVELOPMENT.md

```diff
@@ -0,0 +1,71 @@
+# Using Rust in TensorBoard
```

Comment:
As discussed, it might be nice to just briefly address how to install Rust itself here for the uninitiated (i.e. "see https://rustup.rs").

Reply:
Done.